### PR TITLE
JWT parser

### PR DESCRIPTION
**Pull request by bastantoine**
### Description

This PR adds a new tool that parses a JWT (*JSON Web Token*) and display its values in a human-readable way.

This tools also display enhanced informations about the claims and values based on some standard well defined fields when possible.

### Additional context

*Note*: this doesn't check the validity of the signature of the JWT. I can add that if you feel that it can be useful, but I'd rather get this first part approved, or not, first.

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Submit the PR against the `dev` branch.
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.

### CODE REVIEW

**Comment by bastantoine** [comment]
Any news @CorentinTh ?

**Comment by bastantoine** [comment]
Ping @CorentinTh 😄

File: src/tools/jwt-parser/jwt-parser.service.ts [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+interface JWT {
+  header: Map<string, unknown>;
+  payload: Map<string, unknown>;
+}
+
+export function safe_jwt_decode(raw_jwt: string): JWT {
+  try {
+    const header = jwt_decode(raw_jwt, { header: true }) as Map<string, unknown>;
+    const payload = jwt_decode(raw_jwt) as Map<string, unknown>;
+    return { header: header, payload: payload };
```
Comment (CorentinTh) [inline src/tools/jwt-parser/jwt-parser.service.ts:12]:
```suggestion
    return { header, payload };
```

**Comment by CorentinTh** [comment]
Hello @bastantoine !
First of all, I'm trully sorry to have taken so long to take the time to review your pr 🙏🏻 

Thank you to have taken the time to contribute, a JWT parser will be a fine adition to the it-tool collection, I'm looking forward to your future contributions ❤️

**Review by CorentinTh — CHANGES_REQUESTED** [review]
Just a few thing before the merge :

* Use camelCase for variables and functions name 
* Always show the complete values (remove the *Display parsed value?* toggle) -> it had nothing to hide some information here
* Please rebase your pr with dev to remove the conflicts

**Comment by CorentinTh** [comment]
Also, this project uses [pnpm](https://pnpm.io/) instead of npm, so can you remove the package-lock.json and reinstall using pnpm please 🙏🏻

**Comment by CorentinTh** [comment]
Thanks for those update !

**Review by CorentinTh — APPROVED** [review]

**Comment by CorentinTh** [comment]
I'll try to put the dev branch to prodution this week 💪🏻

**Comment by bastantoine** [comment]
Great thanks! Joyeuses fêtes 🎉 !

**Pull request merged**